ViewModels/BynderFieldDisplayViewModel.cs

```diff
@@ -1,9 +1,20 @@
 using CSM.Bynder.Fields;
 using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace CSM.Bynder.ViewModels;
 
 public class BynderFieldDisplayViewModel
 {
     public IList<BynderResource> Resources { get; } = new List<BynderResource>();
+
+    public Task AddResourcesAsync(IList<BynderResource> resources)
+    {
+        foreach (var resource in resources)
+        {
+            Resources.Add(resource);
+        }
+
+        return Task.CompletedTask;
+    }

```

Comment:
This should be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)-ed since the same thing is in `BynderFieldEditViewModel.cs`, and almost the same thing in `BynderField.cs`. Maybe create an extension method for them in an `Extensions` folder.